orderbook/src/api/get_orders.rs

```diff
@@ -29,25 +29,35 @@ struct Query {
     include_insufficient_balance: bool,
     #[serde(default)]
     include_unsupported_tokens: bool,
+    page_size: Option<u32>,
+    page: Option<u32>,
 }
 
 impl Query {
     fn order_filter(&self) -> Result<OrderFilter, &'static str> {
         if self.owner.is_none() && self.sell_token.is_none() && self.buy_token.is_none() {
             return Err("need to set at least one of owner, sell_token, buy_token");
         }
-        let to_h160 = |option: Option<&H160Wrapper>| option.map(|wrapper| wrapper.0);
-        Ok(OrderFilter {
-            min_valid_to: self.min_valid_to,
-            owner: to_h160(self.owner.as_ref()),
-            sell_token: to_h160(self.sell_token.as_ref()),
-            buy_token: to_h160(self.buy_token.as_ref()),
-            exclude_fully_executed: !self.include_fully_executed,
-            exclude_invalidated: !self.include_invalidated,
-            exclude_insufficient_balance: !self.include_insufficient_balance,
-            exclude_unsupported_tokens: !self.include_unsupported_tokens,
-            uid: None,
-        })
+
+        return match self.page_size {
+            Some(i) if !(1..=100).contains(&i) => Err("page_size must be between 1 and 100"),
+            _ => {
+                let to_h160 = |option: Option<&H160Wrapper>| option.map(|wrapper| wrapper.0);
+                Ok(OrderFilter {
+                    min_valid_to: self.min_valid_to,
+                    owner: to_h160(self.owner.as_ref()),
+                    sell_token: to_h160(self.sell_token.as_ref()),
+                    buy_token: to_h160(self.buy_token.as_ref()),
+                    exclude_fully_executed: !self.include_fully_executed,
+                    exclude_invalidated: !self.include_invalidated,
+                    exclude_insufficient_balance: !self.include_insufficient_balance,
+                    exclude_unsupported_tokens: !self.include_unsupported_tokens,
+                    uid: None,
+                    page_size: self.page_size,
+                    current_page: self.page,
```

Comment:
Are u defining some defaults?

Probably I would even do the things in this orders:
- First do all validations, u validate `page_size` but not `current page` (unless this is zero based, then I guess the type will validate). 
- Construct the struct

Reply:
Right now I'm using hard coded defaults for page_size (10) and default page (0)
Do intend to clear it up and have proper constants, but not there yet.